### DISCUSSION TITLE
读《中县干部》 | /posts/fjq/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#594] [Announcements]
https://yuanfan.rbind.io/posts/fjq/

**Comment by Kiki-2333-Zhang** [comment]
哈哈哈哈最近也在重温《小镇喧嚣》。对“这狗X的收税”印象深刻。以及其实我们业内有个关于《中县干部》学术伦理的八卦，博主感兴趣的话可以去搜一搜。

**Reply by earfanfan** [reply]
那个学术伦理的争议我之前看到过，不过其实我比较好奇的是为撒十几年过去了，冯都没写出来一本书，也搜不到他近些年的论文。

**Reply by luyuewei** [reply]
五年后就写了《新化复印产业的生命史》，你没搜到不代表他没发表

**Reply by earfanfan** [reply]
@luyuewei  其实我是先看了《新化复印产业的生命史》这篇论文，然后才看的《中县干部》。

**Comment by earfanfan** [comment]
>在这套逻辑之外，还存在其他未解之谜。比如2002年我国有20个省开始实行农村税费改革，改革的重点是减轻农民负担，但2002年的农业税税额却比上一年增加了49%。

今早在周飞舟的《以利为利》中读到了“黄宗羲定律”，又找来了秦晖的《并税式改革与“黄宗羲定律”》看，大约理解了所谓“积累莫返之害”导致我国历史上每一次税制改革短期可行长期又继续加重农民负担。以后若有时间，应当多找些各方面的数据来分析看看。
